.github/workflows/prod.yml

```diff
@@ -0,0 +1,25 @@
+name: 'Push to Github Packages'
+
+on:
+    pull_request:
+        types: closed
+        branches:
+            - master
+
+jobs:
+    push_to_registry:
+        name: Push Docker image to GitHub Packages
+        runs-on: ubuntu-latest
+        steps:
+            - name: Check out the repo
+              uses: actions/checkout@v2
+            - name: Push to GitHub Packages
+              uses: docker/build-push-action@v1
+              with:
+                  username: ${{ github.actor }}
+                  password: ${{ secrets.GITHUB_TOKEN }}
+                  registry: docker.pkg.github.com
+                  # Uppercase letters are not supported so this is all lowercase
+                  repository: ncats-gamma/robokache/build
+                  tags: latest
```

Comment:
Can we also tag specific versions when we make releases?

Reply:
I updated the code to have two separate workflows for deployment. Pushes to main will be tagged with the commit hash as well as the tag `main`, and releases will be tagged with the semantic version as well as the `latest` tag. I don't have an easy way of debugging this code because I can't run a release on my local machine. If you could create a release to test the code that would be very helpful (maybe just tag it with some old version like `v0.1.0`).